### PR TITLE
[AN-540] : [VentIntegration] - Phase 1 complete

### PR DESCRIPTION
**Pull request by sambuds**
This commit updates postgres db with all necessary tables to shadow legacy sqlsol.

Signed-off-by: Sam Sinha <ssinha484@gmail.com>

### CODE REVIEW

File: contracts/src/bpm-model/DefaultProcessModelRepository.sol [hunk 1]
```diff
@@ -63,10 +63,31 @@ contract DefaultProcessModelRepository is Versioned(1,0,0), AbstractEventListene
 		// of new Process Model address. Bug: https://plan.monax.io/issue/AN-299
 		// Until it's sorted out we shall not use emitEvent as a way to trigger the ProcessModelRepo to push a sqlsol update.
 		// Instead the ProcessModelRepo will directly issue the push after a new model is created
-		_model.addEventListener("UPDATE_PROCESS_MODELS", this);
+		_model.addEventListener("paa-S", this);
```
Comment (j-h-scheufen) [inline contracts/src/bpm-model/DefaultProcessModelRepository.sol]: what does paa-S stand for?

File: contracts/src/bpm-runtime/ApplicationRegistry.sol [hunk 2]
```diff
@@ -14,6 +14,33 @@ contract ApplicationRegistry is Upgradeable {
...
+		bytes32 indexed eventId,
+		bytes32 application_id,
+		uint8 application_type,
+		address location,
+		bytes4 method,
+		bytes32 web_form,
+		uint access_point_count
+	);
+
+	event LogApplicationAccessPointCountUpdate(
```
Comment (j-h-scheufen) [inline contracts/src/bpm-runtime/ApplicationRegistry.sol]: It seems an event to update the count might be a relic from the old sqlsol days. Do we need to keep the count in the sql DB as a column if a query can take care of it?

File: contracts/src/bpm-model/DefaultProcessModelRepository.sol (hunk 1 above)
Comment (sambuds) [reply contracts/src/bpm-model/DefaultProcessModelRepository.sol]: Updated

File: contracts/src/bpm-runtime/ApplicationRegistry.sol (hunk 2 above)
Comment (sambuds) [reply contracts/src/bpm-runtime/ApplicationRegistry.sol]: Added TODO comment

**Pull request merged**